### DISCUSSION TITLE
EV3 PyBricks cannot pair with Spike Prime 3

### DISCUSSION THREAD

**Discussion by gfairSG** [pybricks/support#1651] [General]
Hi Helpful PyBrick Gurus,

I couldn't find any discussions about new issues with pairing an EV3 to an updated Spike Prime 3.  It seems like LEGO removed some pairing functionality with their latest release for Spike.  Does anyone know of any workarounds?

We are attempting to setup an EV3 as a remote control for Spike bots to make them more exciting to middle schoolers for fun competitions.  We've been following Anton's blog 
[https://www.antonsmindstorms.com/2021/09/02/reuse-your-old-ev3-brick-to-remote-control-your-new-spike-prime-or-robot-inventor-hub/](https://www.antonsmindstorms.com/2021/09/02/reuse-your-old-ev3-brick-to-remote-control-your-new-spike-prime-or-robot-inventor-hub/) 
to do this and were able to get PyBricks up and running along with run the tank code but we simply cannot pair the EV3 with the Spike to finish the connection.

Thank you for any help or direction to go.  Gordon

**Comment by dlech** [comment]
Are you running Pybricks on the SPIKE hub, the EV3 or both?

**Reply by Gordon-Fair** [reply]
Running Pybricks only on EV3.  I don't need any code running on the Spike.  I just need to pair the EV3 to the Spike.

**Reply by dlech** [reply]
You could use Pybricks code to downgrade the SPIKE firmware to v2 official LEGO firmware. But you would also need the older SPIKE app after that to restore the file system on the external flash memory on the SPIKE hub.

**Comment by Gordon-Fair** [comment]
Any help would be much appreciated.

I've downgraded the SPIKE firmware and EV3 was able to see hub  
but
**_it connects for a few seconds and then disconnects._**

I've used SPIKE Legacy to updated firmware within SPIKE Legacy App.  Still same issue with connect.

I've installed PyBricks firmware on SPIKE. Still same issue with connect.

I've tried different EV3s with the micro SD Card.  
I've tried different SPIKEs.

All connect for a second or two and then disconnect with an error not available at this time.

**Reply by Bigbootyfish** [reply]
@dlech Can you find a fix for this entire bluetooth connectivity issue?
